comic_wishlist/routes.py

```diff
@@ -0,0 +1,7 @@
+from comic_wishlist import app
+
+
+
+@app.route('/')
+def index():
+    return 'hellooooooooo'
```

Comment:
you did not put a new line at the end of the file... and why did you just make it return hellooooooo, you should have created a template or something to get us started... in fact, you shoukld create a ticket to create the base layout of the site, import bootstrap, create a .css file for us to use plz and then push so i can see the updates!

Reply:
Added a new line.
Created template.
imported bootstrap and created css